rslearn/models/swin.py

```diff
@@ -141,6 +144,7 @@ def forward(
                 process.
             targets: target dicts that are ignored unless
         """
+        # TODO: Are Targets  supposed to be ignored? Is this to maintain a common interface?
```

Comment:
Hm I think this was so that the module could be used both as an "encoder" in MultiTask / SingleTask or as the full classification model, but really for classification it would make sense to pair it (with output_layers=None) with ClassificationHead so I think just taking inputs makes sense since in that case it would still be used as "encoder" (which is just passed the inputs). So I'll change this.